### ISSUE TITLE
There isn't any auto-complete available in BIDMach and BIDMat unlike scala 

### ISSUE THREAD

**Issue by serioussam1988** [BIDData/BIDMach#113]
There isn't any auto-complete available in BIDMach and BIDMat unlike scala , is that because i am using BIDMach in terminal? can i enable that with installing bundle with Eclipse or something else?

**Comment by kevinkilroy** [comment]
Looks like a bug with the scala REPL and trying to pre-load the file with BIDMat/BIDMach imports in (lib/bidmach_init.scala).
There is a work around, open bidmach, at the bottom of the file remove:
`-i ${LIBDIR}/bidmach_init.scala`
Now run ./bidmach
Obviously you'll need to manually import the packages you want (have a look in lib/bidmach_init.scala if not sure) but at least tab completion will work.

I'll raise a bug over at the scala project as this is still present in 2.11.8

**Comment by kevinkilroy** [comment]
Looks like the scala -i option has been fixed in 2.12.
You can use :load to load the file after ./bidmach has started.
i.e.
```
./bidmach
Welcome to Scala version 2.11.2 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_77).
Type in expressions to have them evaluated.
Type :help for more information.

scala> :load lib/bidmach_init.scala
```
After the file loads all the imports, tab completion will work
